Set/Game/AspectVGrid.swift

```diff
@@ -16,7 +17,7 @@ struct AspectVGrid<Item, ItemView>: View where ItemView: View, Item: Identifiabl
             ScrollView (showsIndicators: false) {
                 VStack {
                     let minWidthOfCard: CGFloat = 95
-                    let width: CGFloat = widthThatFits(itemCount: items.count,
+                    let width: CGFloat = widthThatFits(itemCount: expectedItemsCount,
```

Comment:
Here is easy fix, just make 
- `var expectedItemsCount: Int?` in class definition, so it's not required property
- here `widthThatFits(itemCount: expectedItemsCount ?? items.count,`
- add `expectedItemsCount` in contractor (also optional)
- change this value what it's required